nemo/export/trt_llm/converter/model_to_trt_llm_ckpt.py

```diff
@@ -238,6 +239,246 @@ def handle_model_level_weights(model, tp_idx: int, pp_idx: int):
     return weights_dict
 
 
+def _get_layer_index(split_key):
+    for index, key in enumerate(split_key):
+        if key == "layers":
+            return index + 1
+    raise ValueError(f"Unknown layer name format: {split_key}")
+
+
+def rename_layer_num(param_name, layer_num):
+    split_key = param_name.split(".")
+    layer_index = int(_get_layer_index(split_key))
+    split_key[layer_index] = str(layer_num)
+    return ".".join(split_key)
+
+
+def get_layer_num(param_name):
+    split_key = param_name.split(".")
+    layer_index = int(_get_layer_index(split_key))
+    return int(split_key[layer_index])
+
+
+@torch.no_grad()
+def dist_model_to_trt_llm_ckpt(
```

Comment:
Can dist_model_to_trt_llm_ckpt function and convert_model_to_trt_llm_ckpt be merged into one function for instance?